main/webapp/modules/core/scripts/index/open-project-ui.js

```diff
@@ -149,7 +154,20 @@ Refine.OpenProjectUI.prototype._renderProjects = function(data) {
          console.log('Project '+project.id+' name is not set. skipping...');
          continue;
       }
-      project.date = moment(project.modified).format('YYYY-MM-DD HH:mm A');
+      // project.modified is ISO 8601 format
+      const date = new Date(project.modified);
+      const dateParts = dateFormatter.formatToParts(date);
+      const timeParts = timeFormatter.formatToParts(date);
+      const findPart = (parts, type) => parts.find(part => part.type === type)?.value;
+
+      const year = findPart(dateParts, 'year');
+      const month = findPart(dateParts, 'month');
+      const day = findPart(dateParts, 'day');
+      const hour = findPart(timeParts, 'hour');
+      const minute = findPart(timeParts, 'minute');
+      const dayPeriod = findPart(timeParts, 'dayPeriod'); // AM or PM
+
+      project.date = `${year}-${month}-${day} ${hour}:${minute} ${dayPeriod}`;
```

Comment:
```suggestion
      project.date = dateFormatter.format(date);
```

I suggest we let the internationalization code do *all* of the formatting